fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -476,9 +476,6 @@ public SerializedMetaSpec getSerializedMetaSpec(String dbName, String tableName,
 
         List<ManifestFile> deleteManifests = snapshot.deleteManifests(nativeTable.io());
         List<ManifestFile> matchingDeleteManifests = filterManifests(deleteManifests, nativeTable, predicate);
-        for (ManifestFile file : matchingDeleteManifests) {
```

Comment:
why remove these? how to knoew the matching delete manifest for BE

Reply:
Currently we only support remote plan data file.  The delete file planning is local.